resources/views/tasks/preview.blade.php

```diff
@@ -267,7 +271,8 @@ class="card border-0"
                 is_self_service: 0,
               })
               .then(response => {
-                window.location.reload();
+                window.ProcessMaker.alert('The task was successfully claimed', 'primary', 5, true);
```

Comment:
Please use a trasnslation like:
`window.ProcessMaker.alert(this.$t ('The task was successfully claimed'), 'primary', 5, true);`